InTheMiddle2/InTheMiddle2/ViewController.swift

```diff
@@ -200,12 +203,12 @@ extension ViewController: GMSAutocompleteViewControllerDelegate {
         let camera = GMSCameraPosition.camera(withLatitude: place.coordinate.latitude, longitude: place.coordinate.longitude, zoom: 10.25)
```

Comment:
To add the longitude and latitude into the tableviewcontroller, you may want to create a label text and reference it label to equal the longitude and latitude separately. This will make it more efficiently instead of doing it programmatic.